src/webapp/main.py

```
@@ -0,0 +1,85 @@
+    action: str
+    quantity: int
+    price: float
+
+
+@app.get("/")
+async def root():
+    return {"message": "AI Hedge Fund API"}
+
+
+@app.get("/prices/{ticker}", response_model=List[PriceSchema])
+async def read_prices(ticker: str, db: Session = Depends(get_db)):
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)